build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/ProjectExtensions.kt

```diff
@@ -19,7 +19,22 @@ package com.google.samples.apps.nowinandroid
 import org.gradle.api.Project
 import org.gradle.api.artifacts.VersionCatalog
 import org.gradle.api.artifacts.VersionCatalogsExtension
+import org.gradle.api.file.Directory
+import org.gradle.api.provider.Provider
 import org.gradle.kotlin.dsl.getByType
 
 val Project.libs
     get(): VersionCatalog = extensions.getByType<VersionCatalogsExtension>().named("libs")
+
+/**
+ * Create new Directory by [dir] name in the root build directory.
+ */
+internal fun Project.relativeToRootProject(dir: String): Provider<Directory> =
+    rootProject.layout.buildDirectory.dir(projectDir.toRelativeString(rootDir))
```

Comment:
This will break project isolation. Please use `isolatedProject.rootProject...`

Reply:
Done.